htdocs/cron/list.php

```diff
@@ -239,6 +239,18 @@
 
 llxHeader('', $pagetitle);
 
+$sqlTest = 'SELECT DISTINCT test FROM '.MAIN_DB_PREFIX.'cronjob';
+$resultTest = $db->query($sqlTest);
+$TTestNotAllowed = array();
+if($resultTest) {
+	while ($objTest = $db->fetch_object($resultTest)) {
+		$veriftest = verifCond($objTest->test);
+			if (!$veriftest) {
+				$TTestNotAllowed[] = $db->escape($objTest->test);
```

Comment:
If goal is to exclude some record later, we should store an id into array instead of field test.

Reply:
Why do you prefer id ? If some lines have the same test we'll pass multiple time in verif cond with same test

Reply:
Because later you use the array TTestNotAllowed to exclude some line. And what identify a line is its ID.